mgmt/rest/metric.go

```diff
@@ -41,6 +41,15 @@ func (s *Server) getMetrics(w http.ResponseWriter, r *http.Request, _ httprouter
 }
 
 func (s *Server) getMetricsFromTree(w http.ResponseWriter, r *http.Request, params httprouter.Params) {
+	// we land here if the request contains a trailing slash, because it matches the tree
+	// lookup URL: /v1/metrics/*namespace.  If the length of the namespace param is 1, we
+	// redirect the request to getMetrics.  This results in GET /v1/metrics and
+	// GET /v1/metrics/ behaving the same way.
+	if len(params.ByName("namespace")) <= 1 {
+		s.getMetrics(w, r, params)
+		return
+	}
+
 	ns := parseNamespace(params.ByName("namespace"))
```

Comment:
probably add a var for params.ByName("namespace") instead of calling it twice. Good work!